issues.txt

```diff
@@ -83,10 +110,20 @@ Feeling 'stuck' with cards
 	Army is now classed as a building:
 		No longer:
 			Corrupt: Steal an army from play
-			Lifesteal: Destroy an army, heal yourself by 1
```

Comment:
where did lifesteal go

Reply:
lifesteal still there we are not changing it